docs/overview/README.md

```diff
@@ -0,0 +1,58 @@
+<!--ONE-DOCS-HIDE start-->
+[👈 Return to README](../README.md)
+<!--ONE-DOCS-HIDE end-->
+
+# Welcome to One App
+
+As applications start to scale, and you start adding more developers to the project and new teams are created, it becomes increasingly difficult to release with confidence and to effectively collaborate without requiring large amounts of coordination and communication between teams. 
+
+One App helps solve this problem by providing a framework to build quality, modularized web applications. It helps you break the "Frontend Monolith" and split your application via independently developed, tested, and deployed **Micro Frontends** in the form of **Holocron Modules**.
+
+# Benefits
+
+The benefits to using One App include:
+
+- **Micro Frontends:** Powered by Holocron, One App allows for code splitting via independently developed, tested, and deployed Modules.
+- **Decoupled Codebases:** Each Holocron module has its own repository, reducing large and overwhelming codebases and helping developers to understand the code better. This also avoids accidental coupling by setting specific boundaries.
+- **Security:** Configurable security settings, all pages are protected by a content security policy and all modules are injected with sub-resource integrity.
+- **Flexibility:** An omnichannel solution to UI. Whether you're performing serverside, clientside, delivering modules as a service to web email or phone/IoT clients. One App covers this for you.
+- **Scalability:** Wrapped within an application that scales across 1,000's of engineers and multiple units of an enterprise website.
+- **Reusability:** Modules built as independent user experiences can be easily reused by multiple applications.
+
+
+# Architectural Overview
+
+The One App ecosystem draws a clear line between the App and the React components that make up the user interface. It allows for the React component tree to be split up into modular pieces so that they can be developed and deployed independently. We refer to these modular pieces as… **Modules**. It is an overloaded term, but we wanted to really drive home the point that when thinking about how to design your Modules, they should be as independent and self-contained as possible. The App is responsible for setting up the global redux store with middleware and reducers that all Modules will have access to, and kicking off the rendering of these Modules. [Holocron](https://one-amex-docs.americanexpress.com/en-us/holocron/api/) is the integral library that enables the App and Modules to be separate during development and come back together during production.
```

Comment:
```suggestion
The One App ecosystem draws a clear line between the App and React components that make up the user interface. It allows for the React component tree to be split up into modular pieces so that they can be developed and deployed independently. We refer to these modular pieces as… **Modules**. It is an overloaded term, but we wanted to really drive home the point that when thinking about how to design your Modules, they should be as independent and self-contained as possible. The App is responsible for setting up the global redux store with middleware and reducers that all Modules will have access to, and kicking off the rendering of these Modules. [Holocron](https://one-amex-docs.americanexpress.com/en-us/holocron/api/) is the integral library that enables the App and Modules to be separate during development and come back together during production.
```